ui/apps/platform/src/Containers/Integrations/IntegrationTiles/ClusterRegistrationSecretsTile.tsx

```diff
@@ -9,15 +10,8 @@ import IntegrationTile from './IntegrationTile';
 const { image, label } = descriptor;
 
 function ClusterRegistrationSecretsTile(): ReactElement {
-    const [numIntegrations, setNumIntegrations] = useState(0);
-
-    useEffect(() => {
-        fetchClusterRegistrationSecrets()
-            .then(({ response: { items } }) => {
-                setNumIntegrations(items.length);
-            })
-            .catch(() => {});
-    }, []);
+    const { data } = useRestQuery(fetchClusterRegistrationSecrets);
```

Comment:
Is it worthwhile to do error handling here? I noticed if `data` isn't set, we display a 0. Could be confusing whether it's a 0 or there was an error in fetching the data. We probably aren't doing that for the other tiles though, so I'm ok with deferring that change.

Reply:
This is a good point, I'll leave it for now since it fits in with the current pattern. This would be a good candidate for things to revisit though once we update integrations to not use sagas for the requests.